GitUI/UserControls/IGitBlameParser.cs

```diff
@@ -0,0 +1,8 @@
+﻿using GitUIPluginInterfaces;
+
+namespace GitUI.UserControls;
+
+internal interface IGitBlameParser
```

Comment:
Igor told that internal interfaces with only one implementation should be kept in the same file as the implementation. I agree.